.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: node_js
 node_js:
   - '0.10'
+  - '0.12'
```

Comment:
@phated Do we want to specify '0.10', '0.12', '4' throughout the related repositories?

And as I mentioned elsewhere it seems that some repositories, such as this one, are not triggering a travis build despite having a `.travis.yml`. Any idea why that is?